### PR TITLE
Add license maven plugin

### PR DESCRIPTION
**Pull request by Creepow**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -262,6 +262,20 @@
...
+			
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>license-maven-plugin</artifactId>
+				<version>2.0.0</version>
+				<executions>
+					<execution>
+          				<id>download-licenses</id>
+            			<goals>
+              				<goal>download-licenses</goal>
```
Comment (roesslerj) [inline pom.xml]: Can you fix the formatting?

**Comment by beatngu13** [comment]
Just a short question: Does this run automatically, is the goal coupled to a certain Maven lifecycle? If no, everything's fine. Otherwise, I think we should deactivate it (also in the other PRs) to make sure we do not slow down the build by downloading licenses all the time.

**Comment by Creepow** [comment]
As far as i know it does run automatically and yes it slows down the build by A LOT.

**Comment by beatngu13** [comment]
Then please deactivate everywhere, also in the closed source repos (in case you have added it there).

**Pull request closed without merging**